app/models/driver.rb

```diff
@@ -0,0 +1,38 @@
+class Driver < ApplicationRecord
+  has_many :trips, dependent: :destroy
+
+  validates :name, presence: true, format: {with: /[a-zA-Z]/}
+  validates :vin, presence: true
+
+  def total_earning
+    trips = Trip.where(driver_id: id)
+    sum = 0.00
+    trips.each do |trip|
+      sum += trip.cost
+    end
+    sum = sum * 0.85
+    return sum
+  end
+
+  def average_rating
+    ratings = Trip.where(driver_id: id)
+    result = 0
+    counter = 0
+    ratings.each do |r|
+      if r.rating > 0
+        result += r.rating
+        counter += 1
+      end
+    end
+    if counter > 0
+      return (result/counter)
+    else
+      return "no rating"
```

Comment:
It is fine to return a string here - but is there a way you could normalize the returned data type and provide a numeric result that would represent "no rating"?